twitter-server/src/com/treode/twitter/finagle/http/package.scala

```diff
@@ -125,14 +173,20 @@ package object http {
         TxClock.parse (value) .getOrThrow (new BadRequestException (s"Bad time for $name: $value"))
       }
 
+	//Maintaining compatibility with un-updated servers that use older headers
```

Comment:
Largely this change looks good. However, I think we don't want "maintain compatibility." Did you run into problems without the flexibility? All of this is fairly new, so there aren't many "old style" clients to maintain compatibility for. Moreover, we control all of those "old style" clients, so we can change them. Assuming that we can remove the getOrElse parts below, we need to change the name of these functions to match the new headers.